robottelo/ui/hosts.py

```diff
@@ -71,6 +71,13 @@ def _configure_hosts_parameters(self, parameters_list):
                 if 'active' in inherit.get_attribute('class'):
                     self.click(inherit)
             self.assign_value(param_locator, parameter_value)
+            if parameter_name == 'Memory':
+                # trigger a change event
+                memory_element = self.wait_until_element(param_locator)
+                self.browser.execute_script(
+                    "arguments[0].dispatchEvent(new Event('change'));",
```

Comment:
Do we have some kind of agreed "# TODO" comment or smth to get rid of such blocks after switching to new selenium+geckodriver+ff?

Reply:
we have no agreed, and think it does not depend on selenium version as if you try to change that value by JS, we have the same behavior